README.md

```diff
@@ -8,7 +8,7 @@ Interface for running HUnit tests via Cabal 1.16.
 git clone ... && cd ... && cabal install
 ~~~
 
-Requires HUnit 1.2.* and Cabal 1.16.*
+Requires HUnit 1.2.* and Cabal >= 1.16
```

Comment:
Gotcha. We'll maybe we just remove this then. Real version bounds are in the cabal file where they belong, not need to duplicate them here.